Makefile

```diff
@@ -41,6 +41,8 @@ endif
 
 all: build e2e
 
+pre-commit: update all verify test
```

Comment:
nit: verify already calls update, right?

Reply:
Whoops, missed that. I pushed an update.